packages/ia-components/sandbox/audio-players/youtube-player.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+// No restrictions for using iframe :)
+// functions and parameters such as autoplay, playVideo(), loadVideoById() won't work in all mobile environments
+
+class YoutubePlayer extends Component {
+
+  constructor(props) {
+    super(props);
+
+    const { selectedTrack, id } = props
+
+    this.state = {
+      playerAnchor: React.createRef(),
+      selectedTrack,
+      id
+    }
+
+    // use static methods?
+    this.loadAPI = this.loadAPI.bind(this);
+    this.loadPlayer = this.loadPlayer.bind(this);
```

Comment:
I would prefer to see methods bound to the instance's context using a loop of some sort, like:
```javascript
[
  'loadPlayer',
  'onPlayerStateChange',
  'onPlayerReady',
  'onPlayerError'
].forEach((method) => this[method] = this[method].bind(this));
```